website/docs/components/code-block/partials/guidelines/guidelines.md

```diff
@@ -69,6 +69,21 @@ Line numbers are displayed by default and can make longer blocks of code and sni
 In the Figma component, the code examples have the appropriate number of lines by default but must be manually hidden or shown to match the length of custom snippets.
 !!!
 
+## Height toggle button
+
+When displaying longer code snippets, it can be helpful to let users expand and collapse the content. Setting a `maxHeight` on the Code Block limits how much is shown by default. If the content exceeds this height, a **Show more code** button automatically appears at the bottom of the block, allowing users to expand it. Clicking the button again collapses the block back to its original height.
```

Comment:
Couple of suggestions here:
- Consistent use of "interacting" vs clicking
- Might be helpful to say _why_ it's helpful to the user to expand/collapse the component
- "…will be displayed…" is a bit more formal/active

```suggestion
When displaying longer code snippets, it can be helpful to let users expand and collapse the content to save space in the UI. Setting a `maxHeight` on the Code Block limits how much is shown by default. If the content exceeds this height, a **Show more code** button will be displayed at the bottom of the block, allowing users to expand it. Interacting with the button again collapses the block back to its original height.
```